testing/checkimage.py

```diff
@@ -61,6 +61,11 @@ def check_result_image(fname, baselinefname, threshold, baseline = False, cleanu
     else:
         baselinefnames = [baselinefname,]
 
+    for b in baselinefnames:
```

Comment:
Let's take this bit out -- The VTK image testing backend will pick up on the missing file and print an error while making sure the new baseline still gets uploaded to CDash.